pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_spatial_sweep.py

```diff
@@ -126,49 +197,70 @@ def test__call__(self, mock_inf, mock_foi, mock_poisson, mock_inf_list,
         time = 1
         Parameters.instance().infection_radius = 1000
 
-        test_pop = self.pop
+        test_pop_only1 = self.pop_only1
         test_sweep = SpatialSweep()
 
         # Assert a population with one cell doesn't do anything
-        test_sweep.bind_population(test_pop)
+        test_sweep.bind_population(test_pop_only1)
         test_sweep(time)
         self.assertTrue(self.cell_inf.person_queue.empty())
 
         # Add in another cell with a susceptible, but still
         # no infectors so no infection events.
-        test_pop.add_cells(1)
-        cell_susc = test_pop.cells[1]
-        cell_susc.add_microcells(1)
-        microcell_susc = cell_susc.microcells[0]
-        microcell_susc.add_people(1)
-        infectee = microcell_susc.persons[0]
-        mock_inf_list.return_value = [infectee]
-        mock_list_covid.return_value = [infectee]
+        mock_inf_list.return_value = [self.infectee]
+        mock_list_covid.return_value = [self.infectee]
 
-        test_sweep(time)
-        self.assertTrue(cell_susc.person_queue.empty())
+        test_sweep = SpatialSweep()
 
+        # Assert a basic population
+        test_pop = self.pop
+        test_sweep.bind_population(test_pop)
+        test_sweep(time)
+        self.assertTrue(self.cell_inf.person_queue.empty())
         # Change infector's status to infected
         self.infector.update_status(InfectionStatus.InfectMild)
         test_sweep(time)
-        self.assertEqual(cell_susc.person_queue.qsize(), 1)
+        self.assertEqual(self.cell_inf.person_queue.qsize(), 0)
         Parameters.instance().do_CovidSim = True
-        cell_susc.person_queue = Queue()
+        self.cell_susc.person_queue = Queue()
         test_sweep(time)
-        self.assertEqual(cell_susc.person_queue.qsize(), 1)
-
+        self.assertEqual(self.cell_susc.person_queue.qsize(), 1)
         # Check when we have an infector but no infectees
-        infectee.update_status(InfectionStatus.Recovered)
-        cell_susc.person_queue = Queue()
+        self.infectee.update_status(InfectionStatus.Recovered)
+        self.cell_susc.person_queue = Queue()
         test_sweep(time)
-        self.assertEqual(cell_susc.person_queue.qsize(), 0)
+        self.assertEqual(self.cell_susc.person_queue.qsize(), 0)
 
         # Test parameters break-out clause
         Parameters.instance().infection_radius = 0
         test_sweep(time)
         mock_inf_list.assert_not_called
         mock_list_covid.assert_not_called
-        self.assertEqual(cell_susc.person_queue.qsize(), 0)
+        self.assertEqual(self.cell_susc.person_queue.qsize(), 0)
+
+    @mock.patch("pyEpiabm.sweep.SpatialSweep.find_infectees_Covidsim")
+    @mock.patch("pyEpiabm.sweep.SpatialSweep.find_infectees")
+    @mock.patch("numpy.random.poisson")
+    @mock.patch("pyEpiabm.property.SpatialInfection.space_foi")
+    @mock.patch("pyEpiabm.property.SpatialInfection.cell_inf")
```

Comment:
When you have this many mocks it is generally good practice to check that they are being called the number of times you expect, and with the arguments that you expect (if appropriate)

Reply:
None of the previous tests in this file had it

Reply:
Okay but it might be a good thing to add here!

Reply:
Done now